README.md

```diff
@@ -1,51 +1,163 @@
-# Polyfill service
+Polyfill service
+=====================
+
+Makes web development less frustrating by selectively polyfilling just what the browser needs. Use it on your own site, or as a service.
+
+For usage information see the [hosted service](polyfill-service), which formats and displays the service API documentation located in the [docs](docs/) folder.
 
 [![Build
-Status](https://circleci.com/gh/Financial-Times/polyfill-service.svg?&style=shield&circle-token=357956eb8e6bea4ae9cca8f07918b7d0851a62d1)](https://circleci.com/gh/Financial-Times/polyfill-service)
+Status](https://circleci.com/gh/Financial-Times/polyfill-service.svg?&style=shield&circle-token=357956eb8e6bea4ae9cca8f07918b7d0851a62d1)][ci]
+[![CC0 licensed](https://img.shields.io/badge/license-CC0-blue.svg)][license]
+
+
+Table Of Contents
+-----------------
+
+  * [Requirements](#requirements)
+  * [Running Locally](#running-locally)
+  * [Configuration](#configuration)
+  * [Testing](#testing)
+  * [Release Process](#release-process)
+  * [Deployment](#deployment)
+  * [Publishing](#publishing)
+  * [Monitoring](#monitoring)
+  * [Library API Reference](#library-api-reference)
+  * [License](#license)
+
+
+Requirements
+------------
+
+Running Origami Build Service requires a few tools:
+  * [Git]: For downloading the source code
+  * [Node.js] 5.x and [npm] 3.x: For installing the dependencies and running the application (npm is installed with Node.js)
+  * [Grunt] 0.1.x: Used for automating tasks such as testing
+
+Running Locally
+---------------
+
+Clone the project to our system:
```

Comment:
Generally the README is now quite confused over the use of 'we' and 'our'.  Particularly here, where the system concerned is the personal environment of the reader, so 'our' makes it sound like the clone is being done in some kind of cloud service operated by the polyfill service maintainers.  I suggest using 'you' and 'your' to refer to the reader, and 'we' and 'our' to refer to the maintainers.